### PR TITLE
Add recipes (abandon this PR)

### PR DESCRIPTION
**Pull request by OutOfScopeia**
Create a docker image:
This is done, apart from the last 'experimental' part of the recipe - which already had a caveat above it that it's not guaranteed to work. It's probably just this Dockerfile command that needs more fine-tuning:
RUN dotnet fable src/Client --run npx vite build

Add build automation:
This is a minimal template one, I have only updated Webpack cmd to a Vite cmd.

Add a NuGet package to the Client:
Tested, works without change.

### CODE REVIEW

**Comment by mattgallagher92** [comment]
Is there a reason that you haven't removed the corresponding v4 recipes?

File: docs/recipes/build/add-build-script.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+```
+
+#### 3. Add the project to the solution
+
+Run the following command
+
+```bash
+dotnet sln add Build.fsproj
+```
+#### 4. Installing dependencies
```
Comment (mattgallagher92) [inline docs/recipes/build/add-build-script.md:102]: We should probably include the paket commands for installing the dependencies in this section.

File: mkdocs.yml
Comment (mattgallagher92) [inline mkdocs.yml:1]: Is there a reason to keep the v4 versions of the migrated recipes?

File: docs/recipes/build/docker-image.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+FROM build as server-build
+COPY src/Shared src/Shared
+COPY src/Server src/Server
+RUN cd src/Server && dotnet publish -c release -o ../../deploy
+
+
+FROM build as client-build
+COPY package.json package-lock.json ./
+RUN npm install
+COPY src/Client/vite.config.mts ./
```
Comment (mattgallagher92) [inline docs/recipes/build/docker-image.md:124]: Why do we need to do this? Can't it just stay in src/Client and be copied along with the rest of that directory?

File: docs/recipes/build/add-build-script.md (hunk 1 above)
Comment (mattgallagher92) [reply docs/recipes/build/add-build-script.md:102]: Could be done in a separate PR.

**Comment by martinbryant** [comment]
Closed due to separating the changes into smaller PR's

**Pull request closed without merging**